<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CowMarket</class>
 <widget class="QMainWindow" name="CowMarket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1081</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{

	background-color:rgba(117,78,0,0);
	border-radius:25px;
}
QPushButton:hover{

background-color:rgba(107,66,10,0.4);
}

QLabel{
	
	color:#f0c07d;

}
QMessageBox QPushButton{
	padding:10px;
	padding-left:40px;
	padding-right:40px;

}
QMessageBox{
	border-radius:12px;
	background-color:#0b9e7c;
	font-size:17px;
	font-weight: bold;
}
QMessageBox QLabel{
color:#d1a732;

}
QMenubar::item{
background-color:red;

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>380</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>240</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>400</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>80</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>12</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
